workflow/scripts/compile_xlsx_report.py

```
@@ -0,0 +1,682 @@
+    format_dict = parse_format(fmt, sample)
+
+    # no default here to easier check if CSQ exists
+    csq_data = info_dict.get("CSQ")
+    csq_dict = {}
+    if csq_data:
+        # take the first annotation
+        first_annotation = csq_data.split(",")[0]
+        csq_values = first_annotation.split("|")
+        # check if you want to collect more VEP values than actually exist
+        if len(vep_fields) > len(csq_values):
+            raise ValueError(f"Too may VEP fields are requested: VCF line doesn't have that many")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)